recipe_scrapers/sallysblog.py

```diff
@@ -1,67 +1,43 @@
-        return image_url
-
-    def total_time(self):
-        heading = self.soup.find("p", string="Zubereitungszeit")
-        timing = heading.find_next_sibling("h6")
-        return get_minutes(timing.text)
-
-    def _servings_heading(self):
-        return self.soup.find("h4", string="Zutaten für:")
-
-    def yields(self):
-        servings_heading = self._servings_heading()
-        servings = servings_heading.find_next_sibling("div").find("input")
-        return servings["value"]
-
-    def _groupings(self):
-        servings_heading = self._servings_heading()
-        ingredients_area = servings_heading.next_sibling.next_sibling
-        return ingredients_area.find_all("div", recursive=False)
-
     def ingredients(self):
-        descriptions = []
-        for grouping in self._groupings():
-            ingredients = grouping.find_all("div", recursive=False)
-            for ingredient in ingredients:
-                descriptions.append(ingredient.text)
-
-        return [normalize_string(description) for description in descriptions]
-
-    def instructions(self):
-        grouping_titles = {grouping.find("h5").text for grouping in self._groupings()}
-        uppercase_titles = self.soup.find_all("h2", {"class": "uppercase"})
-
-        descriptions = []
-        for title in uppercase_titles:
-            if title.text in grouping_titles or title.text.endswith("fertigstellen"):
-                instructions = title.find_next_sibling("div").find_all("p")
-                for instruction in instructions:
-                    descriptions.append(instruction.text)
-
-        return "\n".join(
-            [normalize_string(description) for description in descriptions]
+        ingredients = []
+        groupings = self.soup.find_all(
+            "div",
+            {
+                "class": "shop-studio-recipes-recipe-detail-tabs-description-ingredients__content__ingredient-list"
+            },
         )
+        for grouping in groupings:
+            ingredient_items = grouping.find_all(
+                "div",
+                {
+                    "class": "shop-studio-recipes-recipe-detail-tabs-description-ingredients__content__ingredient-list__ingredient"
+                },
+                recursive=False,
+            )
+            for ingredient in ingredient_items:
+                quantity = ingredient.find(
+                    "span",
+                    {
+                        "class": "shop-studio-recipes-recipe-detail-tabs-description-ingredients__content__ingredient-list__ingredient__quantity"
+                    },
+                ).text
+                title = ingredient.find(
+                    "span",
+                    {
+                        "class": "shop-studio-recipes-recipe-detail-tabs-description-ingredients__content__ingredient-list__ingredient__title"
+                    },
+                ).text
+                ingredients.append(f"{quantity} {title}")
```

Comment:
A minor stylistic note: I'd suggest removing the trailing line-end `.text` attribute accesses, and moving them into the f-string instead:

```suggestion
                quantity = ingredient.find(
                    "span",
                    {
                        "class": "shop-studio-recipes-recipe-detail-tabs-description-ingredients__content__ingredient-list__ingredient__quantity"
                    },
                )
                title = ingredient.find(
                    "span",
                    {
                        "class": "shop-studio-recipes-recipe-detail-tabs-description-ingredients__content__ingredient-list__ingredient__title"
                    },
                )
                ingredients.append(f"{quantitytext} {title.text}")
```